tests/Identifier/Normalizer/DateTimeIdentifierNormalizerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Identifier\Normalizer;
+
+use ApiPlatform\Core\Identifier\Normalizer\DateTimeIdentifierDenormalizer;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DateTimeIdentifierNormalizerTest extends TestCase
+{
+    /**
+     * @expectedException \ApiPlatform\Core\Exception\InvalidIdentifierException
```

Comment:
https://thephp.cc/news/2016/02/questioning-phpunit-best-practices

Reply:
interesting, we're using the annotation everywhere though so I guess it's fine :)

Reply:
We could do that in another PR to change it all at one go. :smile:

Reply:
I know about this article, it's interesting, but if we change, we must change everywhere in int the lowest maintained branch.

Reply:
It's not a problem to change it, right? Not a BC break.

Reply:
Indeed. Just some work that is not top priority for me :D